### ISSUE TITLE
[v0.6.5]minimum height is not character height

### ISSUE THREAD

**Issue by bo-ku-ra** [terrastruct/d2#1918]
v0.6.3 no problem.
occur in v0.6.4 and v0.6.5

https://play.d2lang.com/?script=ZMi9CsIwFIbh_VzFS0dBsOt3Nyc_JAW1xUYohNy7k1PWJ-yXWBa6Qfls6R735_f1PsX6l-KHeBi4cPpZ_cii5asNgyDChFHECZNIE95Er3krtYl12LBfAAAA__8%3D&

**Comment by alixander** [comment]
Yes this is by design, otherwise you would never be able to do this: https://play.d2lang.com/?script=qrBSyMnPS1fISUxKzVFIyy9SKEgtKs7P46rQK85ILEi1QvDLM1NKMqwUzAy4AAEAAP__&

**Comment by bo-ku-ra** [comment]
hm.
i want a specification to fit the height of text.

```d2
box: "" {
  grid-columns: 1
  grid-gap: 0
  a: "a" {shape: text}
  b: "b\nb" {shape: text}
  c: "c\nc\nc" {shape: text}
  d: "d\nd\nd\nd" {shape: text}
  *: {height: fitting} 
}
```
